products/plugins/commercial/custom-pricing.md

```diff
@@ -0,0 +1,32 @@
+# Custom Pricing
+
+The Custom Pricing feature allows massive advances in the pricing model capabilities in the shopware 6 ecosystem.
+
+The API interface exposed by this module allows the user to operate a set of commands which allow for the granular 'overriding' of prices via an external data repository or ERP system. This is achieved by defining a custom relationship between a existing price and one of the following entities:
+
+- Customer
+
+<!-- These price->entity relationships will come later
+- Customer Group 
+- Group Id
+- Rule Id
+-->
+
+## API usage
+
+The Custom Pricing API interface models itself upon the interface of the sync API, therefore you will be able to package your requests in a similar manner, see our [API documentation](https://shopware.stoplight.io/docs/admin-api).
```

Comment:
Is it "similar" enough so that referring to the Admin API is sufficient? I think we should comment on the differences if there are any and also refer to the sync API documentation (Bulk Payloads) directly.

Reply:
@elkmod not really, no, there are some key differences
Do you think we should write a guide?

Reply:
Anything that shows the difference would be useful I think. Maybe a guide in the Admin API documentation that we can refer to? Or a section within the "Bulk Payloads" chapter explaining how to use the Custom Pricing API in sufficient detail.

Reply:
@sushmangupta - what do you think about this?

Reply:
Yes, infact, we can describe it in both sections (Guide and Concepts) with different contexts.
With the inclusion of the Admin API quick-start guide, the Guides and the Concepts sections are restructured. Then "Bulk Payloads" are going to be part of the concepts section.

- Now we can add a sub-topic under "Bulk Payloads", Concepts section to explain it and the key differences. 

- Also, another sub-topic under the Guides section to detail how it is handled.

Reply:
Sounds good to me 👍

Reply:
I agree that perhaps a guide about the distinct differences between the Sync API and our endpoint may be useful, we will work on this, but I would say that a concepts document about the sync API should be referred to the respective team who published the Sync API, IMHO

Reply:
I did the first draft, implementing your feedback. Feedback's appreciated. 💙

Reply:
@elkmod Can this thread be resolved?